nextflow.config

```diff
@@ -49,9 +49,9 @@ params {
   // Variant Calling
   ascat_ploidy = null // Use default value
   ascat_purity = null // Use default value
-  cf_coeff = "0.015"  // default value for Control-FREEC
+  cf_coeff = "0.05"  // default value for Control-FREEC
```

Comment:
We are changing it to the default value as recommend in their manual (http://boevalab.inf.ethz.ch/FREEC/tutorial.html). Was there any specific reason to use a value different than the default one?

Reply:
Well, our colleagues played with that, and were happier with 0.05. Will ask why.

Reply:
I see :) I personally believe that is nice to have as default the same as in Control-FREEC specially given that this parameter can be changed by the user. The optimal value is probably different for every case but of course I may be wrong here.

Reply:
As I check the Control-FREEC source, this value is used only at one place: https://github.com/BoevaLab/FREEC/blob/master/src/GenomeCopyNumber.cpp#L60 and the sole purpose is to calculate the windowSize, if it is not provided in the config. For the clarity, it is actually `genome_size/(read_number*cf_coeff^2)` and my guess the readlength should be considered somewhere too, but whatever. Smaller windowSize should mean higher resolution but more noise. We can use the default, sure, and change the value in our own config. I can run some tests on our data, to be safe before release.

Reply:
Interesting, the recommended setting for WES is to use cv_window = 0. I added an entry about that in the docs. It may be a good idea to mention this for the cv_coef parameter.